tests/fixtures/file_1.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
same with files, don't replace old files - just add new ones with `nested` in name